pod/perlvar.pod

```diff
@@ -1257,28 +1268,33 @@ surprising.
 This variable was added in Perl v5.10.0. The C<%{^CAPTURE}> alias was
 added in 5.25.7.
 
-This variable is read-only.
+This variable is read-only, and its value is dynamically scoped.
 
 =item @LAST_MATCH_START
 
 =item @-
 X<@-> X<@LAST_MATCH_START>
 
+This array holds the offsets of the beginnings of the last successful
+match and any capture buffers it contains.
+(See L</Scoping Rules of Regex Variables>).
+
+The number of elements it contains will be one more than the number of
+the highest capture buffers (also called a subgroup) that actually
+matched something. (As opposed to C<@+> which may have less elements.)
```

Comment:
strict English grammar is 'fewer elements", though less and less people care

Note typo "addss" in first commit message.

Reply:
I shall fix both so I have fewer typos credited to my name.

Reply:
Done.